back_translation/Backtranslate Using Google API.ipynb

```diff
@@ -0,0 +1,292 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    print(f"Estimated cost of this run is ${total_price}")
</pre><p>This is useful :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SunbirdAI/parallel-text-EDA/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
:money_with_wings: :money_with_wings: